Calculator/Calculator/Model/CalculatorItemQueue.swift

```diff
@@ -0,0 +1,38 @@
+struct CalculatorItemQueue<T>: CalculateItem {
+    private var calculateItems = LinkedList<T>()
+    
+    mutating func enqueue(_ element: T) {
+        calculateItems.insert(data: element)
+    }
+    
+    mutating func showHead() -> (data: T?, next: Node<T>?) {
+        let headData = calculateItems.head?.data
+        let headNext = calculateItems.head?.next
+        return (headData, headNext)
+    }
+    
+    mutating func showTail() -> (data: T?, next: Node<T>?) {
+        let tailData = calculateItems.tail?.data
+        let tailNext = calculateItems.tail?.next
+        return (tailData, tailNext)
+    }
+    
+    mutating func dequeue() -> T? {
+        if isEmpty() {
+            return nil
+        }
+        return calculateItems.delete()
+    }
+    
+    mutating func isEmpty() -> Bool {
+        if calculateItems.count == 0 {
+            return true
+        } else {
+            return false
+        }
+    }
+    
+    mutating func resetQueue(){
+        calculateItems.resetLinkedList()
+    }
```

Comment:
```suggestion
    mutating func reset() {
        calculateItems.reset()
    }
```
=> 네이밍 중복인 것 같아 위처럼 제안드려봅니다. 이미 큐이므로 reset 으로만 네이밍해도 이해 가능하고, 이미 링크드리스트이므로 reset 으로만 네이밍해도 이해 가능해서요!

Reply:
`CalculatorItemQueue` 안에 메서드들은 제안해주신 네이밍으로 수정했습니다!
`LinkedList` 안에 메서드들을 기존 배열 사용시 썼던 메서드명과 동일하게 수정했습니다😌
ex) reset -> removeAll